plugins/notifications/email/main.go

```diff
@@ -88,7 +89,7 @@ func (n *EmailPlugin) Notify(ctx context.Context, notification *protobufs.Notifi
 	logger.Debug("smtp connection done")
 
 	email := mail.NewMSG()
-	email.SetFrom(fmt.Sprintf("From <%s>", cfg.SenderEmail)).
+	email.SetFrom(fmt.Sprintf("%s <%s>", cfg.SenderName, cfg.SenderEmail)).
```

Comment:
Just to not break existing installations, setting a default `SenderName` would be nice. Either here or even better in the `Configure` method right after unmarshal. 

```go
if cfg.SenderName == "" {
   cfg.SenderName = "CrowdSec" 
}
```

Reply:
Yes that's a good remark, but as i had not seen any config input validation or default settings, I had continued like this
it can be clearly improved I agree

Reply:
just implement default settings and validation